### PR TITLE
[REVIEW] Fix dead link in README

### PR DESCRIPTION
**Pull request by awthomp**
Fix dead link in README to Jetson install instructions.

### CODE REVIEW

**Comment by awthomp** [comment]
rerun tests

**Comment by awthomp** [comment]
@BradReesWork can you work your magic to get this PR merged? Looks like CI is stuck.

File: README.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ This code executes on an NVIDIA V100 in 637 ms.
 ## Installation
 
 ### Conda, Linux OS
-cuSignal can be installed with conda ([Miniconda](https://docs.conda.io/en/latest/miniconda.html), or the full [Anaconda distribution](https://www.anaconda.com/distribution/)) from the `rapidsai` channel). If you're using a Jetson GPU, please follow the build instructions [below](https://github.com/rapidsai/cusignal#conda-jetson-nano-tk1-tx2-xavier-linux-os)
+cuSignal can be installed with conda ([Miniconda](https://docs.conda.io/en/latest/miniconda.html), or the full [Anaconda distribution](https://www.anaconda.com/distribution/)) from the `rapidsai` channel). If you're using a Jetson GPU, please follow the build instructions [below](https://github.com/rapidsai/cusignal#conda---jetson-nano-tk1-tx2-xavier-linux-os)
```
Comment (BradReesWork) [inline README.md:100]: You have an extra unneeded ")" after channel

Comment (awthomp) [reply README.md]: Fixed!

**Pull request merged**